### ISSUE TITLE
refgenie id behavior for genome digests

### ISSUE THREAD

**Issue by nsheff** [refgenie/refgenie#222]
Right now, `refgenie id` returns an asset digest. Should it also provide genome digests?

```
refgenie id hg38/fasta
59fed5ec3c3a95bc5709b24b8f2cf7bf
```

but: 
```
refgenie id hg38
Provided asset registry path (None/hg38:None) is invalid. See help for usage reference.
```

Or is there another way to show genome digests?

**Comment by stolarczyk** [comment]
there's a command specifically designed for this:

```
~ refgenie alias get -a hg38
58de7f33a36ccd9d6e3b1b3afe6b9f37cd5b2867bbfb929a
```

**Comment by stolarczyk** [comment]
```
usage: refgenie alias get [-h] [-c C] [--skip-read-lock] [-a A [A ...]]

Get aliases.

optional arguments:
  -h, --help                         show this help message and exit
  -c C, --genome-config C            Path to local genome configuration file. Optional if
                                     REFGENIE environment variable is set.
  --skip-read-lock                   Whether the config file should not be locked for
                                     reading
  -a A [A ...], --aliases A [A ...]  Aliases to get the digests for.
```

**Comment by nsheff** [comment]
yep, I remember now. -- great I like that.  Do you think we should change the `id` terminology then? It is a bit confusing, I think... Like, why is `id` for asset digests and `alias get` for genome digests? maybe we can streamline this a little

**Comment by nsheff** [comment]
Also may be a documentation issue. "Get aliases" does not seem to be to indicate that it will return a digest. I was expecting that to return a list of aliases.

**Comment by stolarczyk** [comment]
yeah, I agree that this might be confusing, but the main purpose of this is `refgenie alias get`, which gives you a list of all aliases and digests defined.

when you specify the alias, then you basically get one row from there, but there's no point in displaying the alias since you've just provided it.

**Comment by nsheff** [comment]
I see. that does make sense. But do you see how that was a bit roundabout way to get there from the perspective of someone just trying to lookup a digest for a particular alias? I wonder if there's another way to simplify it.

**Comment by stolarczyk** [comment]
maybe the command you suggested at the very beginning would do?

to get the digest for hg38 alias:
```
refgenie id hg38
```

to get the digest for hg38/fasta asset:
```
refgenie id hg38/fasta
```

**Comment by nsheff** [comment]
is it confusing at all that one command, `id` would yield two different types of responses depending on its argument? the genome identifier is a DRUID, whereas the asset identifier is just an md5sum.

Maybe not a big deal?

And for `refgenie alias`, it could equivalently be `refgenie digest` since it returns both aliases and digests, right? Is there some other terminology we could use there? Can that be rolled into `refgenie id` ?

**Comment by nsheff** [comment]
@stolarczyk is this resolved?

**Comment by stolarczyk** [comment]
I didn't make any changes since we didn't settle on anything. This is how it works now:

- to get asset digest: `refgenie id <genome>/<asset>`
- to get genome DRUID (completely different from the digest above): `refgenie alias get -a <genome>`
